Libraries/acdriver/Headers/acdriver/Options.h

```diff
@@ -73,6 +74,10 @@ class Options {
     ext::optional<std::string> _filterForDeviceOsVersion;
     ext::optional<std::string> _assetPackOutputSpecifications;
 
+private:
+    // Options not compatible with Apple's actool
+    Extensions::ActoolOptions        _extensions;
```

Comment:
You've got several misalignments going on in this diff; match the file?  _extensions should be aligned with all the other member names.